2021-04-06 10:46:07,459 INFO    MainThread:29118 [internal.py:wandb_internal():62] W&B internal server running at pid: 29118
2021-04-06 10:46:07,461 DEBUG   HandlerThread:29118 [handler.py:handle_request():54] handle_request: check_version
2021-04-06 10:46:07,462 INFO    WriterThread:29118 [datastore.py:open_for_write():76] open: /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/run-2uvumlob.wandb
2021-04-06 10:46:07,464 DEBUG   SenderThread:29118 [sender.py:send():88] send: header
2021-04-06 10:46:07,464 DEBUG   SenderThread:29118 [sender.py:send():88] send: request
2021-04-06 10:46:07,464 DEBUG   SenderThread:29118 [sender.py:send_request():97] send_request: check_version
2021-04-06 10:46:07,469 DEBUG   Thread-4  :29118 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): pypi.org:443
2021-04-06 10:46:08,062 DEBUG   Thread-4  :29118 [connectionpool.py:_make_request():393] https://pypi.org:443 "GET /pypi/wandb/json HTTP/1.1" 200 55912
2021-04-06 10:46:08,266 DEBUG   SenderThread:29118 [sender.py:send():88] send: run
2021-04-06 10:46:08,292 DEBUG   SenderThread:29118 [cmd.py:execute():719] Popen(['git', 'cat-file', '--batch-check'], cwd=/home/menglh/rlproj/tmp/on-policy, universal_newlines=False, shell=None, istream=<valid stream>)
2021-04-06 10:46:08,307 DEBUG   SenderThread:29118 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2021-04-06 10:46:08,781 DEBUG   SenderThread:29118 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-04-06 10:46:08,793 INFO    SenderThread:29118 [dir_watcher.py:__init__():167] watching files in: /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/files
2021-04-06 10:46:08,793 INFO    SenderThread:29118 [sender.py:_start_run_threads():464] run started: 2uvumlob with start time 1617677166
2021-04-06 10:46:08,811 DEBUG   HandlerThread:29118 [handler.py:handle_request():54] handle_request: run_start
2021-04-06 10:46:08,812 DEBUG   SenderThread:29118 [sender.py:send():88] send: summary
2021-04-06 10:46:08,813 INFO    SenderThread:29118 [sender.py:_save_file():542] saving file wandb-summary.json with policy end
2021-04-06 10:46:08,829 DEBUG   HandlerThread:29118 [cmd.py:execute():719] Popen(['git', 'cat-file', '--batch-check'], cwd=/home/menglh/rlproj/tmp/on-policy, universal_newlines=False, shell=None, istream=<valid stream>)
2021-04-06 10:46:08,839 DEBUG   HandlerThread:29118 [cmd.py:execute():719] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=/home/menglh/rlproj/tmp/on-policy, universal_newlines=False, shell=None, istream=None)
2021-04-06 10:46:08,911 DEBUG   SenderThread:29118 [sender.py:send():88] send: files
2021-04-06 10:46:08,912 INFO    SenderThread:29118 [sender.py:_save_file():542] saving file wandb-metadata.json with policy now
2021-04-06 10:46:08,914 DEBUG   HandlerThread:29118 [handler.py:handle_request():54] handle_request: status
2021-04-06 10:46:08,915 DEBUG   SenderThread:29118 [sender.py:send():88] send: request
2021-04-06 10:46:08,915 DEBUG   SenderThread:29118 [sender.py:send_request():97] send_request: status
2021-04-06 10:46:08,925 DEBUG   SenderThread:29118 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2021-04-06 10:46:08,928 DEBUG   Thread-11 :29118 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2021-04-06 10:46:09,244 DEBUG   SenderThread:29118 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-04-06 10:46:09,258 DEBUG   Thread-11 :29118 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-04-06 10:46:09,263 DEBUG   Thread-11 :29118 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): storage.googleapis.com:443
2021-04-06 10:46:09,738 DEBUG   Thread-11 :29118 [connectionpool.py:_make_request():393] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/reinholdm/StarCraft2/2uvumlob/wandb-metadata.json?Expires=1617763569&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=JhvkDlsZgDtO5a80VkxYYP%2Bq1JHfbeg14p3ibnQNvGUUMfHcjQmoq1E7gt2hQNoyXmL0Uv%2FKmdzcQQXOfzALtHVG2I0BqX2DqP%2BPEOrTvzGnK58vy87K8wYy7YMNIZdYDoLw1ekKo9WytOJNbbHPXZGeeEgmycHcx1QqDQWMycCPSIRA77H9LdZDFU77WgDWy6kZXMvF6pf9v52uGLBk1Xzv9SssuFAdpAzUVvfF%2FqoTSDSeE6xwGfi8l3aQaO7r10Ad%2F2QjhwaNbn3Snvs%2F425aLDw%2BYhM0YPwiXIVaiVj90L3OahC7EAbfSkxVVlYBFWYJzwFakEz2YhS9KFshEA%3D%3D HTTP/1.1" 200 0
2021-04-06 10:46:09,741 INFO    Thread-11 :29118 [upload_job.py:push():112] Uploaded file /tmp/tmpmsir_juwwandb/2jbi6jcq-wandb-metadata.json
2021-04-06 10:46:09,797 INFO    Thread-8  :29118 [dir_watcher.py:_on_file_created():215] file/dir created: /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/files/requirements.txt
2021-04-06 10:46:09,798 INFO    Thread-8  :29118 [dir_watcher.py:_on_file_created():215] file/dir created: /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/files/wandb-summary.json
2021-04-06 10:46:09,798 INFO    Thread-8  :29118 [dir_watcher.py:_on_file_created():215] file/dir created: /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/files/output.log
2021-04-06 10:46:09,799 INFO    Thread-8  :29118 [dir_watcher.py:_on_file_created():215] file/dir created: /home/menglh/rlproj/tmp/on-policy/onpolicy/scripts/results/StarCraft2/corridor/mappo/mlp/wandb/run-20210406_104606-2uvumlob/files/wandb-metadata.json
2021-04-06 10:46:10,815 DEBUG   Thread-5  :29118 [connectionpool.py:_new_conn():823] Starting new HTTPS connection (1): api.wandb.ai:443
2021-04-06 10:46:11,023 WARNING MainThread:29118 [internal.py:wandb_internal():116] Internal process interrupt: 1
2021-04-06 10:46:11,124 DEBUG   Thread-5  :29118 [connectionpool.py:_make_request():393] https://api.wandb.ai:443 "POST /files/reinholdm/StarCraft2/2uvumlob/file_stream HTTP/1.1" 200 None
2021-04-06 10:46:11,519 WARNING MainThread:29118 [internal.py:wandb_internal():116] Internal process interrupt: 2
2021-04-06 10:46:11,520 ERROR   MainThread:29118 [internal.py:wandb_internal():119] Internal process interrupted.
2021-04-06 10:46:11,630 INFO    MainThread:29118 [internal.py:handle_exit():137] Internal process exited
